<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrickGame (by burshlat)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #fcba03;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frPanel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>290</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    background-color: gray;
    border: 3px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="gvField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView {
	border: 1px solid #303030;
	background-color: #555555;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbScore">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 1px solid black;
	background-color: #4d4e4f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>High score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbHighScore">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 1px solid black;
	background-color: #4d4e4f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbLevel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 1px solid black;
	background-color: #4d4e4f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Next:</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="gvNextFigure">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView {
	border: 1px solid #303030;
	background-color: #555555;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSpeed">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
	border: 1px solid black;
	background-color: #4d4e4f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Snake</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tetris</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbWalls">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>5</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Walls</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No walls</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>171</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause(P)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sound(S)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start(R)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnPause">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 9px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid green;
}

QPushButton:pressed {
  background-color: white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSound">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 9px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid green;
}

QPushButton:pressed {
  background-color: white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #8F8F8F;
	border-radius: 9px;
	border: 3px solid gray;
	color: #82D625;
}

QPushButton:hover {
    border: 3px solid red;
}

QPushButton:pressed {
  background-color: white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnReset">
           <property name="font">
            <font>
             <pointsize>35</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 3px solid gray;
    border-radius: 25px;
    background-color: #4CAF50;
    background-color: #8F8F8F;
}

QPushButton:hover {
    border: 3px solid blue;
}

QPushButton:pressed {
  background-color: white;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Reset(SPACE)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>141</width>
      <height>157</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QFrame {
	border: 0px black;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnRight">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid gray;
    border-radius: 19px;
    background-color: #4CAF50;
    background-color: #8F8F8F;
}

QPushButton:hover {
    border: 3px solid blue;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnDown">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid gray;
    border-radius: 19px;
    background-color: #4CAF50;
    background-color: #8F8F8F;
}

QPushButton:hover {
    border: 3px solid blue;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnLeft">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid gray;
    border-radius: 19px;
    background-color: #4CAF50;
    background-color: #8F8F8F;
}

QPushButton:hover {
    border: 3px solid blue;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="btnUp">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid gray;
    border-radius: 19px;
    background-color: #4CAF50;
    background-color: #8F8F8F;
}

QPushButton:hover {
    border: 3px solid blue;
}

QPushButton:pressed {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Up /</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rotate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
